replicate_manifest_config.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+if [ "$1" == "-h" ] || [ "$1" == "--help" ] || [ "$#" -ne 2 ]; then
+  echo "------------------------------------------------------------------------------"
+  echo "Usage - replicate_manifest_config <source_manifest> <target_manifest>"
+  echo ""
+  echo "e.g., ./replicate_manifest_config.sh internalstaging.datastage.io/manifest.json gitops-qa/jenkins-dcp.planx-pla.net/manifest.json"
+  echo ""
+  echo "The script pulls data from a source manifest and replaces the versions & dictionary of the target manifest"
+  echo "-------------------------------------------------------------------------------"
+  exit 0;
+fi;
+
+svcs=(arborist fence indexd aws-es-proxy peregrine pidgin revproxy sheepdog portal fluentd spark tube manifestservice wts guppy sower hatchery ambassador)
```

Comment:
you can get a list of services with `jq` if you want
```
jq -r -e '.versions | keys'
```
something like that

Reply:
That's neat! Let me push another commit with this change.